app/dashboard/page.tsx

```diff
@@ -0,0 +1,67 @@
+"use client";
+import React, { FC, useEffect, useState } from "react";
+import Navbar from "../components/Header";
+import accountHelper from "../helpers/account";
+import { Account } from "stellar-sdk";
+
+const Dashboard: FC = () => {
+  const [isFunded, setIsFunded] = useState(false);
+  const [publicKey, setPublicKey] = useState("" as string);
+  const [isLogged, setIsLogged] = useState(publicKey !== "");
```

Comment:
Why not just start with `true` as an initial value?

Reply:
I changed the default value of isFunded to true.